packages/nullable-array/nullable-array.0.1/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+name: "ocplib-nullable-array"
+version: "0.1"
+maintainer: "Pierre Chambart <pierre.chambart@ocamlpro.com>"
+authors: "Pierre Chambart <pierre.chambart@ocamlpro.com>"
+homepage: "https://github.com/chambart/ocplib-nullable-array"
```

Comment:
The URL I found is `ocaml-` not `ocplib-` : https://github.com/chambart/ocaml-nullable-array